server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/WebAdminServerIntegrationTest.java

```diff
@@ -498,6 +504,29 @@ public void reprocessingAllTaskShouldCreateATask() throws Exception {
                 .body("taskId", is(Matchers.notNullValue()));
     }
 
+    @Test
+    public void reprocessingOneTaskShouldCreateATask() throws Exception {
```

Comment:
does it test that the task is serialized?

Reply:
yes, but anyway I will improve the tests to be more coherent